lightning/src/chain/chaininterface.rs

```diff
@@ -41,12 +41,13 @@ pub enum ConfirmationTarget {
 pub trait FeeEstimator {
 	/// Gets estimated satoshis of fee required per 1000 Weight-Units.
 	///
-	/// Must be no smaller than 253 (ie 1 satoshi-per-byte rounded up to ensure later round-downs
-	/// don't put us below 1 satoshi-per-byte).
+	/// Must return a value no smaller than 253 (ie 1 satoshi-per-byte rounded up to ensure later  
+	/// round-downs don't put us below 1 satoshi-per-byte).
 	///
-	/// This translates to:
-	///  * satoshis-per-byte * 250
-	///  * ceil(satoshis-per-kbyte / 4)
+	/// This translates to taking the max of the fee rate multipled by 250 (1 sat/vbyte) e.g
```

Comment:
They're all feerates, just different units. You aren't multiplying the feerate by 250, you're multiplying certain units by 250.